Tests/PubNubTests/Networking/Routers/SpaceObjectsRouterTests.swift

```diff
@@ -29,32 +29,27 @@
 import XCTest
 
 // swiftlint:disable:next type_body_length
-final class SpaceObjectsEndpointTests: XCTestCase {
+final class SpaceObjectsRouterTests: XCTestCase {
   let config = PubNubConfiguration(publishKey: "FakeTestString", subscribeKey: "FakeTestString")
-  let fetchAllSpaces = Endpoint.objectsSpaceFetchAll(include: nil, limit: nil, start: nil, end: nil, count: nil)
-  let fetchSpace = Endpoint.objectsSpaceFetch(spaceID: "SomeSpace", include: nil)
   let testSpace = SpaceObject(name: "TestSpace")
   let invalidSpace = SpaceObject(name: "")
+}
 
-  // MARK: - Fetch All Tests
+// MARK: - Fetch All Tests
 
-  func testFetchAll_Endpoint() {
-    let endpoint = Endpoint.objectsSpaceFetchAll(include: .custom, limit: 100, start: "Start", end: "End", count: true)
+extension SpaceObjectsRouterTests {
+  func testFetchAll_Router() {
+    let router = SpaceObjectsRouter(.fetchAll(include: .custom, limit: 100, start: "Start", end: "End", count: true), configuration: config)
 
-    XCTAssertEqual(endpoint.description, "Fetch All Space Objects")
-    XCTAssertEqual(endpoint.category, .objectsSpaceFetchAll)
-    XCTAssertEqual(endpoint.operationCategory, .objects)
-    XCTAssertNil(endpoint.validationError)
+    XCTAssertEqual(router.endpoint.description, "Fetch All Space Objects")
+    XCTAssertEqual(router.category, "Fetch All Space Objects")
+    XCTAssertEqual(router.service, .objects)
   }
 
-  func testFetchAll_Endpoint_AssociatedValues() {
-    let endpoint = Endpoint.objectsSpaceFetchAll(include: .custom, limit: 100, start: "Start", end: "End", count: true)
+  func testFetchAll_Router_ValidationError() {
+    let router = SpaceObjectsRouter(.fetchAll(include: .custom, limit: 100, start: "Start", end: "End", count: true), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 140 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)